daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -1896,6 +1896,27 @@ class RazerBlackWidowV4(_RippleKeyboard):
     DEVICE_IMAGE = "https://dl.razerzone.com/src2/13219/13219-1-en-v1.png"
 
 
+class RazerBlackWidowV4X(_RippleKeyboard):
+    """
+    Class for the Razer BlackWidow V4 X
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Razer_BlackWidow_V4_X(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0293
+    HAS_MATRIX = True
+    WAVE_DIRS = (1, 2)
+    MATRIX_DIMS = [6, 22]
+    METHODS = ['get_device_type_keyboard', 'set_wave_effect', 'set_wheel_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_ripple_effect', 'set_ripple_effect_random_colour']
+
+    DEVICE_IMAGE = "https://m.media-amazon.com/images/I/71qoXjgRb-L._AC_SL1500_.jpg"
```

Comment:
Only images from Razer please. I found https://dl.razerzone.com/src2/13223/13223-1-en-v1.png for this device which looks I think the same so I'm updating it.